### ISSUE TITLE
Error: Command failed: 'unzip' is not recognized as an internal or external command, operable program or batch file.

### ISSUE THREAD

**Issue by capaj** [jspm/jspm-cli#392]
I got this error when running 

    jspm install bootstrap

under windows 10, but I think it will be the same in all Windows.
I went ahead and tried to install unzip from http://gnuwin32.sourceforge.net/packages/unzip.htm and added that to my path.
Then:

```
Error: Command failed: 'chmod' is not recognized as an internal or external command,
     operable program or batch file.
```

Could you maybe use https://github.com/EvanOxfeld/node-unzip instead of command line unzip?

**Comment by guybedford** [comment]
Thanks, I just added a fix for this today in https://github.com/jspm/github/commit/8071129247bdb966dd5d7a8fae10f538f83cb1c2. Release will be following shortly.

**Comment by capaj** [comment]
so this fix is going to be in 0.10.5?

**Comment by guybedford** [comment]
Yes, working on it.

**Comment by capaj** [comment]
@guybedford just a question about the bug processes- wouldn't it make more sense to close the issue when the release containing the fix gets released? And I am not criticising the speed at which you fix, because, you are insanely fast. It is only that this way when someone googles the error and finds the closed issue, he might think that the issue has been resolved. He would have to actually look at the commit and look at package.json to realize that this hasn't been resolved yet in the released version of jspm-cli.
So to put it simply- I think bugs should stay open, until the fix is released on npm. Do you agree? It is also great for people who are watching this issue, that they'll get a mail notification when the fix is available.

**Comment by guybedford** [comment]
@capaj that makes sense as a suggestion - I'll see if I can get a label workflow for this worked out.
